.github/release.yml

```diff
@@ -0,0 +1,20 @@
+changelog:
+  exclude:
+    authors:
+      - github-actions
+  categories:
+    - title: Features
+      labels:
+        - "type: feature"
+    - title: Bugs
+      labels:
+        - "type: bug"
+    - title: Documentation
+      labels:
+        - "type: docs"
+    - title: Chores
+      labels:
+        - "type: chore"
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
Where those labels must be specified in the pr? is it in the change set?

Reply:
Labels on the PR